src/server/game/Miscellaneous/Language.h

```diff
@@ -876,8 +876,9 @@ enum TrinityStrings
     LANG_SQLDRIVER_QUERY_LOGGING_ENABLED  = 1027,
     LANG_SQLDRIVER_QUERY_LOGGING_DISABLED = 1028,
     // 1029-1030 used in other client versions
-
     // Room for more level 4              1031-1099 not used
+    LANG_ACCOUNT_PASS_TOO_LONG            = 1031,
+
```

Comment:
How its related to the PR?

Reply:
No it was my bad, i've created the PR with my fork on master branch, now this PR is merged than this snippet will not be considered.